lib/http.c

```diff
@@ -1388,6 +1395,37 @@ static int http_getsock_do(struct connectdata *conn,
   return GETSOCK_WRITESOCK(0);
 }
 
+static void add_haproxy_protocol_header(struct connectdata *conn, bool *done)
+{
+  char proxy_header[128];
+  Curl_send_buffer *req_buffer;
+  CURLcode result;
+  char tcp_version[5];
+
+  /* Emit the correct prefix for IPv6 */
+  if (conn->bits.ipv6) {
+    strcpy(tcp_version, "TCP6");
+  } else {
```

Comment:
We don't write 'else' clauses like that: (both these style nits were found with `make checksrc` )
~~~
./http.c:1410:3: warning: else after closing brace on same line (BRACEELSE)
   } else {